src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--3">
+      <div class="calendar__day calendar__day--1 calendar__day--sun"></div>
+      <div class="calendar__day calendar__day--2"></div>
```

Comment:
tag with `calendar__day` selector shouldn't have extra classes
```suggestion
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
```